### PR TITLE
Remove one deprecated API

### PR DESCRIPTION
**Pull request by tzm41**
- Remove one deprecate API: `openURL`.
- Bump deployment target to iOS 10. From now on newer version of Intrepid pod will support minimum iOS 10.

### CODE REVIEW

File: SwiftWisdom/Core/UIKit/UIApplication+Extensions.swift [hunk 1]
```diff
@@ -17,6 +17,6 @@ extension UIApplication {
         guard let url = URL(string: UIApplication.openSettingsURLString), canOpenURL(url) else {
             throw OpenSettingsError.cannotOpenURL
         }
-        openURL(url)
+        open(url)
```
Comment (bobgilmore) [inline SwiftWisdom/Core/UIKit/UIApplication+Extensions.swift:20]: Let me investigate a bit and see if this is acceptable for extensions?

**Review by paulrolfe — COMMENTED** [review]
Can this bump to remove support for 9.0 be captured in the readme? Similar to how we deal with swift versions?

**Comment by jpmendel** [comment]
Yeah, I agree with @paulrolfe, we should probably start tracking the iOS version support in the readme. Otherwise this looks good.

**Comment by tzm41** [comment]
@jpmendel @paulrolfe Added a tiny snip in README for iOS version.

**Review by jpmendel — APPROVED** [review]

**Review by bobgilmore — APPROVED** [review]

**Pull request merged**